books/RayTracingTheRestOfYourLife.html

```diff
@@ -135,7 +164,7 @@
 
             if (runs % 100000 == 0)
                 std::cout << "Estimate of Pi = "
-                          << 4*double(inside_circle) / runs
+                          << 4 * double(inside_circle) / runs
```

Comment:
Same question as above. Probably a quick pass on the other utility programs if you agree.

Reply:
Agree.